plans.html

```diff
@@ -1,38 +1,45 @@
 --- 
 layout: default 
-title: OED plans 
+title: OED Release Notes 
 ---
 <div> 
-	<h1>Plans for the Open Energy Dashboard</h1>
-	<p>
-		While OED has been public since March 2017, we have not yet had a
-		major public release. To coincide with AASHE 2017, we will be doing
-		our first release in mid-October. Please see the <a
-			href="features.html">features page</a> to see what that release can
-		do.
-	</p>
-	<p>The OED project already has features we hope to release in 3-6
-		months (January-March 2018). These include:</p>
-	<ul>
-		<li>Generalization of energy types beyond electricity including
-			steam, gas, etc.</li>
-		<li>An admin panel to allow someone responsible for the OED
-			system to import meters, enter information needed, and generally
-			control the way OED functions at their site.</li>
-		<li>Allow energy to be represented in other units such as CO<sub>2</sub>,
-			laundry loads, etc.
-		</li>
-		<li>Have a virtual energy usage page to allow someone to understand
-			the cost associated with using different types of appliances given the
-			time per use, how often and how long used, and the number of people
-			who use those appliances. Note that we are considering stopping development of this feature and pointing people to a very nice virtual room developed by Arizona State University.</li>
-		<li>Make the OED website controls and basic wording
-			multilingual.</li>
-		<li>Provide email warnings of system issues.</li>
-	</ul>
-	<p>
-		We expect to include other features in the next release. You can
-		influence our plans by <a href="involved.html">becoming involved</a>.
-	</p>
-	<p>We also have a list of <a href="future.html">feature requests</a> that we have yet to decide when to implement.</p>
+  <h1>Release Information</h1>
+    <p>For more information on the features discussed on this page see our <a href="features.html">feature page</a>.</p>
+
+
+  <h4>Version 0.2</h4>
+  <ul>
+    <li>Added ability to graph a meter as a line or bar graph.</li>
```

Comment:
I think this should say 0.2 was the first major release that includes the ability to automatically acquire, store in a database and display electric meter data.